resource-management/pkg/store/redis/redis.go

```diff
@@ -37,7 +37,7 @@ type Goredis struct {
 }
 
 const (
-	redisPort = "6379"
+	redisPort = "7379"
```

Comment:
better to fix it along with the other changes, so we are not hard code this anymore.

Reply:
Done.